SDL/Globals.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Globals_h
+#define Globals_h
+
+#include "Constants.h"
+#include "Module.h"
+#include "TiltedGeometry.h"
+#include "EndcapGeometry.h"
+#include "ModuleConnectionMap.h"
+#include "PixelMap.h"
+
+namespace SDL {
+  template <typename>
+  struct Globals;
+  template <>
+  struct Globals<Dev> {
+    static SDL::modulesBuffer<Dev>* modulesBuffers;
+    static SDL::modulesBuffer<Dev> const* modulesBuffersES;  // not owned const buffers
+    static uint16_t nModules;
+    static uint16_t nLowerModules;
+    static std::shared_ptr<SDL::pixelMap> pixelMapping;
+
+    static EndcapGeometry<SDL::Dev>* endcapGeometry;
+    static TiltedGeometry<SDL::Dev> tiltedGeometry;
+    static ModuleConnectionMap<SDL::Dev> moduleConnectionMap;
```

Comment:
I am trying to understand the differences in this part of the code. What is the logic behind the fact that some objects are under the `SDL` namespace and others are not, and then the inverse is true for their template parameter?

Reply:
it's due to a mix of reusing search/replace and copy-paste patterns from different places.